src/main/kotlin/no/skatteetaten/aurora/cantus/ApplicationConfig.kt

```diff
@@ -21,4 +21,9 @@ class ApplicationConfig : BeanPostProcessor {
             it.realmName = "CANTUS"
         }
     }
+
```

Comment:
Vurder kortform på disse:


    fun basic() = BasicAuthenticationEntryPoint()
        .also { it.realmName = "CANTUS" }

    @Bean
    fun restTemplate() = RestTemplate()

Reply:
Lurt! Det ser mye bedre ut.